src/ui/coverage/coverageInfo.ts

```diff
@@ -103,82 +120,68 @@ async function runCoverageCommand(command: string, harnessName: string): Promise
  */
 async function parseKaniCoverageOutput(stdout: string): Promise<any | undefined> {
 	const kaniOutput: string = stdout;
-	const kaniOutputArray: string[] = kaniOutput.split('Coverage Results:\n');
 
-	const coverageResults = kaniOutputArray.at(1);
+	const jsonFilePath = getCoverageJsonPath(kaniOutput);
+	const coverageMap: Map<string, Map<any, string>> = new Map();
+
+	if (jsonFilePath) {
+		const files = fs.readdirSync(jsonFilePath);
+		const jsonFiles = files.filter(file => path.extname(file) === '.json');
+
+		// In jsonFiles, store the string that contains the substring kaniraw.json and read that path
+		if(jsonFiles.length > 0) {
+			const kanirawJson = jsonFiles.find(file => file.includes('kaniraw.json'));
+			if(kanirawJson) {
```

Comment:
```suggestion
			if (kanirawJson) {
```
Is the formatter not catching those issues in CI?

Reply:
https://github.com/model-checking/kani-vscode-extension/pull/146